lib/oban/crontab/scheduler.ex

```diff
@@ -51,6 +51,7 @@ defmodule Oban.Crontab.Scheduler do
 
   @impl GenServer
   def handle_continue(:start, state) do
+    insert_reboot(state.conf)
```

Comment:
Reading back through the initial conversation I see that you were unsure whether this needed to be inside a `lock_and_enquque` block. It should, otherwise there's a chance of inserting duplicate jobs. Additionally, for consistency it would be nice to use `insert` consistently, i.e. `lock_and_insert`, `insert_jobs`.

Reply:
Should it use the same key as well, since one node may be inserting reboot jobs, and another node may be enqueuing scheduled ones? Or is it fine to assume that the node handling reboot jobs will also hit the scheduled ones if the other nodes are locked out due to reboot?

Reply:
It is fine to assume that the node handling reboot jobs will get the scheduled ones as well. Ideally we'd run both forms of `insert` from a single lock transaction.